core/frontend/src/tile/GltfReader.ts

```diff
@@ -2123,8 +2123,15 @@ export abstract class GltfReader {
       return;
 
     try {
+      // Refuse to continue decoding if using Internet Explorer or old Microsoft Edge. We do not want to trigger any legacy decoding fallbacks within draco3d.
+      if (ProcessDetector.isIEBrowser) {
```

Comment:
seems odd we need to add this at all here since we dont support ie at all anyway.

Reply:
Can you direct me to code in iTwin.js that guarantees it will not run in IE?  I found checkbrowser.js, but I think it was only consumed by DTA an DPTA.

Reply:
If you're using vite, you need to copy static assets, and we show how to do so in our examples; [see](https://github.com/iTwin/viewer/blob/master/packages/templates/web/vite.config.mts#L15-L24). That should also copy the `checkbrowser.js` script, its up to the dev to add that script to `index.html`. So looks like we don't explicitly disallow IE in  iTwin.js, but if im not mistaken we now only support webgl2 as of `4.0` and IE doesnt even support webgl2. If anything we should add a check in `itwin/webgl-compatibility`?

Reply:
The default `RenderSystem` uses webgl-compatibility but apps can override it, so you're not guaranteed here that we've already safeguarded against IE. I think just leave the check here; ultimately we're going to delete all this code anyway and let CesiumJS handle glTF decoding.